resources-service/src/import/sql.ts

```diff
@@ -25,12 +25,12 @@ export const generateUpsertSqlFromImportResource = (
   const sqlBatch: string[] = [];
 
   sqlBatch.push(`${pgp.helpers.insert(
-    { ...resourceRecord, accountSid },
-    ['id', 'name', 'accountSid'],
+    { ...resourceRecord, accountSid, created: resourceRecord.updatedAt },
+    ['id', 'name', 'accountSid', 'created'],
```

Comment:
This one will update the created using the `resourceRecord.updatedAt`, right? Why if so?

Reply:
I just thought it best to use the updated date from the API in both cases. If we use the current timestamp we could end up with a created date later than the lastUpdated date, because 'lastUpdated' means when it was updated in the source DB, when 'created' would mean when it was created in our DB. I thought it best to use the 'lastUpdated' as 'created' for new records.

If we get a specific 'created' date from the KHP API IO might raise a quick PR to change it to use that instead. Down the line, if it's useful, we could have a second created/updated pair, to record when it was created / updated locally, but only if we need it